docs/content/config/index.md

```diff
@@ -0,0 +1,79 @@
+HedgeDoc can be configured via environment variables either directly or via an `.env` file.
+
+## The `.env` file
+
+The `.env` file should be places in the root of the project and contains key value pairs of environment variables, and the values you configured them with. This can for example look like this:
+
+```ini
+HD_DOMAIN="http://localhost"
+HD_MEDIA_BACKEND="filesystem"
+HD_MEDIA_BACKEND_FILESYSTEM_UPLOAD_PATH="uploads/"
+HD_DATABASE_DIALECT="sqlite"
+HD_DATABASE_STORAGE="./hedgedoc.sqlite"
+```
+
+We also provide an `.env.example` file containing a minimal configuration in the root of the project. This should help you to write your own configuration.
+
+!!! warning  
+    The minimal configuration provided in `.env.example` is exactly that: minimal. It will let you start HedgeDoc, but it is **not** meant to be used in production without prior changes.
+
+## General
+
+| environment variable     | default   | example                    | description                                                                                       |
+|--------------------------|-----------|----------------------------|---------------------------------------------------------------------------------------------------|
+| `HD_DOMAIN`              | -         | https://md.example.com     | The URL the HedgeDoc instance runs on                                                             |
+| `PORT`                   | 3000      |                            | The port the HedgeDoc instance runs on                                                            |
+| `HD_RENDERER_ORIGIN`     | HD_DOMAIN |                            | The URL the renderer runs on. If omitted this will be same as `HD_DOMAIN`                         |
```

Comment:
```suggestion
| `HD_RENDERER_ORIGIN`     | HD_DOMAIN |                            | The URL the renderer runs on. If omitted this will be same as `HD_DOMAIN`.                        |
```

Reply:
done